app/src/pages/lloydGeorgeRecordPage/LloydGeorgeRecordPage.tsx

```diff
@@ -0,0 +1,116 @@
+import { useBaseAPIUrl } from '../../providers/configProvider/ConfigProvider';
+import getLloydGeorgeRecord from '../../helpers/requests/lloydGeorgeSearchResult';
+import PdfViewer from '../../components/generic/pdfViewer/PdfViewer';
+import { getFormattedDatetime } from '../../helpers/utils/formatDatetime';
+import { DOWNLOAD_STAGE } from '../../types/generic/downloadStage';
+import formatFileSize from '../../helpers/utils/formatFileSize';
+
+function LloydGeorgeRecordPage() {
+    const [patientDetails] = usePatientDetailsContext();
+    const [downloadStage, setDownloadStage] = useState(DOWNLOAD_STAGE.INITIAL);
+    const [numberOfFiles, setNumberOfFiles] = useState(0);
+    const [totalFileSizeInByte, setTotalFileSizeInByte] = useState(0);
+    const [lastUpdated, setLastUpdated] = useState('');
+    const [lloydGeorgeUrl, setLloydGeorgeUrl] = useState('');
+    const navigate = useNavigate();
+    const baseUrl = useBaseAPIUrl();
+
+    const dob: String = patientDetails?.birthDate
+        ? getFormattedDate(new Date(patientDetails.birthDate))
+        : '';
+
+    const nhsNumber: String =
+        patientDetails?.nhsNumber.slice(0, 3) +
+        ' ' +
+        patientDetails?.nhsNumber.slice(3, 6) +
+        ' ' +
+        patientDetails?.nhsNumber.slice(6, 10);
+
+    const patientInfo = (
+        <>
+            <p style={{ marginBottom: 5, fontWeight: '700' }}>
+                {`${patientDetails?.givenName} ${patientDetails?.familyName}`}
+            </p>
+            <p style={{ fontSize: '16px', marginBottom: 5 }}>NHS number: {nhsNumber}</p>
+            <p style={{ fontSize: '16px' }}>Date of birth: {dob}</p>
+        </>
+    );
+
+    useEffect(() => {
+        if (!patientDetails) {
+            navigate(routes.HOME);
+        } else {
+            const search = async () => {
+                setDownloadStage(DOWNLOAD_STAGE.PENDING);
+                const nhsNumber: string = patientDetails?.nhsNumber || '';
+                try {
+                    const { number_of_files, total_file_size_in_byte, last_updated, presign_url } =
+                        await getLloydGeorgeRecord({
+                            nhsNumber,
+                            baseUrl,
+                        });
+                    if (presign_url?.startsWith('https://')) {
+                        setNumberOfFiles(number_of_files);
+                        setLastUpdated(getFormattedDatetime(new Date(last_updated)));
+                        setLloydGeorgeUrl(presign_url);
+                        setDownloadStage(DOWNLOAD_STAGE.SUCCEEDED);
+                        setTotalFileSizeInByte(total_file_size_in_byte);
+                    }
+                    setDownloadStage(DOWNLOAD_STAGE.SUCCEEDED);
+                } catch (e) {
+                    setDownloadStage(DOWNLOAD_STAGE.FAILED);
+                }
+            };
+            void search();
+        }
+    }, [patientDetails, baseUrl, navigate]);
```

Comment:
Does this cause a memory leak? I had to fix the last document search code that looked similar to this search function. 
See DocumentSearchResults for example with using a ref.mounted and toggling it so we know this search just runs once.